ManatiCR/vitaminau
# VIT-44: Update custom field in "Uso de instalaciones" form.

# Task [VIT-44](https://manati.atlassian.net/browse/VIT-44?atlOrigin=eyJpIjoiODY4ODQ4YjUyMDMzNDk5MWIzMWZkZGM4ZDFlMTM5YWUiLCJwIjoiaiJ9)
Creation of the "Uso de instalaciones" form.

#Installation
- Run the following command
`ahoy drush cr && ahoy drush cim -y`

# Test
- Login as admin user:
  - User: admin
  - Pass: admin
- Go to the following url: 
  - http://vitaminau.docker/form/webform-installations-use
- Once on this page, go to the "Equipo especializado" section.
- Try pressing the "+" button to add a new row of items.
- Try pressing the "-" button to delete a row of items.
- Verify that the changed field is the same as the Jira ticket.
  - [VIT-44](https://manati.atlassian.net/browse/VIT-44?atlOrigin=eyJpIjoiODY4ODQ4YjUyMDMzNDk5MWIzMWZkZGM4ZDFlMTM5YWUiLCJwIjoiaiJ9)
- Fill all the fields of the form and press the save buttom
- Once you press it your information should be saved and the form should redirect you to a thank you page

# Acceptance criteria
- I can add multiple data sets.
- I can add it and I don't have errors.
- "Ficha técnica" is required.
